README.md

```diff
@@ -27,7 +27,7 @@ You can install the example chrome extension from [Chrome Web Store](https://chr
 
 ### Why it exists
 
-Due to limitations in the Chrome extension environment regarding executing remote code, the FingerprintJS Pro JavaScript agent doesn't work there by default.
+The Chrome extension environment has limited capabilities for external scripts running within content scripts. This repository showcases different strategies on how FingeprintJS Pro can be used in this ecosystem. 
```

Comment:
```suggestion
The Chrome extension environment has limited capabilities for external scripts running within it. This repository showcases different strategies on how FingeprintJS Pro can be used in this ecosystem. 
```

I would change it to something like this, because the limited capabilities are in every part of the extension, not only in the content scripts.

Reply:
@TheUnderScorer thanks, I will rephrase it a little bit.